t/mojo/request.t

```diff
@@ -475,6 +475,11 @@ subtest 'Parse full HTTP 1.0 request (no scheme and empty elements in path)' =>
   is $req->url,                     '//foo/bar//baz.html?foo=13', 'right URL';
   is $req->headers->content_type,   'text/plain',                 'right "Content-Type" value';
   is $req->headers->content_length, 27,                           'right "Content-Length" value';
+  is $req->headers->host,           undef,                        '"Host" value is not defined';
+
+  $req->fix_headers;
+  is $req->url->host,     undef, 'still no url host';
```

Comment:
A nitpick, but I see that the rest of the file is mostly has alignment of the expected values and the test description. Perhaps align these two as well?

Reply:
I'll change `url` to `URL`, but the formatting is purely what perltidy produced.